File: app/src/main/java/zone/com/zrefreshlayoutdemo/common/HeadSetting.java
package zone.com.zrefreshlayoutdemo.common;

import zone.com.zrefreshlayout.ZRefreshLayout;

/**
 * MIT License
 * Copyright (c) [2018] [Zone]
 */

public class HeadSetting {
    public static final int METERIAL = 0;
    public static final int SINA = 1;
    public static final int WAVE = 2;

    private int headmode;
    private ZRefreshLayout.HeadPin headPin = ZRefreshLayout.HeadPin.NOT_PIN;

    public int getHeadmode() {
        return headmode;
    }

    public void setHeadmode(int headmode) {
        this.headmode = headmode;
    }

    public ZRefreshLayout.HeadPin headPin() {
        return headPin;
    }

    public void setPin(ZRefreshLayout.HeadPin pin) {
        this.headPin = pin;
    }
}

File: app/src/main/java/zone/com/zrefreshlayoutdemo/common/HeadSetting.kt
package zone.com.zrefreshlayoutdemo.common

import zone.com.zrefreshlayout.ZRefreshLayout

/**
 * MIT License
 * Copyright (c) [2018] [Zone]
 */

class HeadSetting {

    var headmode: Int = 0
    private var headPin: ZRefreshLayout.HeadPin = ZRefreshLayout.HeadPin.NOT_PIN

    fun headPin(): ZRefreshLayout.HeadPin {
        return headPin
    }

    fun setPin(pin: ZRefreshLayout.HeadPin) {
        this.headPin = pin
    }

    companion object {
        val METERIAL = 0
        val SINA = 1
        val WAVE = 2
    }
}
